CODING_STANDARD

```diff
@@ -1,17 +1,31 @@
 Here a few minimalistic coding rules for the cprover source tree:
 
-a) 2 spaces indent, no tabs
-b) no "using namespace std;"
-c) Avoid new/delete, use containers instead.
-d) Avoid unnecessary #includes, especially in header files
-e) No lines wider than 80 chars
-f) Put matching { } into the same column
-g) If a method is bigger than a page, break it into parts
-h) Avoid destructive updates if possible. The irept has
-   constant time copy.
+- Use 2 spaces indent, no tabs.
+- No lines wider than 80 chars.
```

Comment:
The code base, by and large, actually uses 70 chars. Though be sensible about it: don't trade extra-short (and then meaningless) variable names for achieving short lines.